doxia-site-renderer/pom.xml

```diff
@@ -168,7 +172,7 @@ under the License.
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>1.10.19</version>
+      <version>2.18.3</version>
```

Comment:
We do dependency updates in separate tickets.

Reply:
I've updated this in https://github.com/apache/maven-doxia-sitetools/pull/7

Reply:
Please create also a new ticket for this.